content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -150,13 +150,13 @@ RuntimeClass Beta (v1.14) included the following changes:
   meaning it can no longer container `.` characters (in all versions). Valid handlers match the
   following regular expression: `^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`.
 
-If you were using RuntimeClass Alpha (prior to v1.14), the following actions are required
+If you were using RuntimeClass Alpha (prior to v1.14), the following **actions are required**:
```

Comment:
nit: line 150, s/container/contain ?

Reply:
nit: line 140, possible edit:
The RuntimeClass Beta feature includes the following changes:

line 143: s/CustomResourceDefintion/CustomResourceDefinition